view_all_inc.php

```diff
@@ -111,10 +111,15 @@
 		<div class="btn-toolbar">
 			<div class="btn-group pull-left">
 		<?php
-			# -- Print and Export links --
+			# -- Print link --
```

Comment:
I don't see a reason to treat it in another way than the other buttons, see http://www.mantisbt.org/bugs/view.php?id=22224#c55574